lib/qunit-bdd.js

```diff
@@ -68,6 +68,7 @@
           try {
             fn.call(env);
           } catch (e) {
+            e.stack && console.warn(e.stack);
```

Comment:
What happens if we use `e.stack` instead of `QUnit.config.current.stack` in the `pushFailure` call?